lib/l10n_utils/fluent.py

```diff
@@ -0,0 +1,90 @@
+from functools import wraps
+from hashlib import sha256
+
+from django.conf import settings
+from django.core.cache import caches
+from django.utils.encoding import force_bytes
+
+from fluent.runtime import FluentLocalization, FluentResourceLoader
+
+
+cache = caches['l10n']
+
+
+class FluentL10n(FluentLocalization):
+    _localized_messages = None
+
+    def _localized_bundles(self):
```

Comment:
@Pike perhaps this is a good way to go for the `FluentLocalization` class? Then you can do things like below.